api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -586,6 +586,12 @@ public EventType get(final String eventTypeName) throws NoSuchEventTypeException
         return eventType;
     }
 
+    public EventType getNoCache(final String eventTypeName) throws NoSuchEventTypeException, InternalNakadiException {
+        final EventType eventType = eventTypeRepository.findByName(eventTypeName);
+        authorizationValidator.authorizeEventTypeView(eventType);
```

Comment:
that was not there before, why do we need it on that level?

Reply:
Trying to mimic `get()` a few lines above which has this auth check.